packages/angular/projects/clr-angular/src/forms/datepicker/date-container.ts

```diff
@@ -66,12 +66,7 @@ import { IfControlStateService, CONTROL_STATE } from '../common/if-control-state
           ></clr-datepicker-view-manager>
         </div>
         <clr-icon class="clr-validate-icon" shape="exclamation-circle"></clr-icon>
-        <clr-icon
-          *ngIf="valid && controlSuccessComponent"
-          class="clr-validate-icon"
-          shape="check-circle"
-          aria-hidden="true"
-        ></clr-icon>
+        <clr-icon *ngIf="valid" class="clr-validate-icon" shape="check-circle" aria-hidden="true"></clr-icon>
```

Comment:
I'm not seeing a View Basics spec that tests the presence or absence of this element based on state. Since this is done based on the service and not `isValid` I think it needs to have coverage.